odpf/meta/facets/Tags.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package odpf.meta.facets;
+
+option java_package = "io.odpf.proton.meta.facets";
+option java_outer_classname = "Tags";
+option go_package = "github.com/odpf/proton/meta/facets";
+
+message Tags {
```

Comment:
Do we really need a facet for `Tag`? Do you see in the near future anyone adding any more fields other than this map?

Reply:
hmm good question 🤔 , as of now I havent found our use cases yet.
The tag facet is inspired from evaluating how Datahub and Amundsen modelling their metadata.

Im fine right now if we want to remove tag facet. wdyt @ravisuhag ?

Reply:
Yeah. should be fine.